docs/en/current/settings/export.md

```diff
@@ -0,0 +1,22 @@
+# Export Settings 
+
+K-9 provides the ability to import and export your settings. This is useful when changing/adding a device.
+
+The settings are exported as a file, stored in a location of your choice on the device.
+
+You can now move this file to a new device or just keep it there when you uninstall and then re-install K-9 Mail on the
+same device. The settings file does not contain any of your passwords.
+
+To export your settings, go to Settings -> Backup -> Export settings
+
+![Import/Export Menu](img/settings_import_export_menu.png)
+
+By default you will export all your settings, including all account-specific settings. If you want to
+export only the settings for a particular account, deselect the other accounts.
+
+![Import/Export Menu](img/settings_export_select_accounts.png)
+
+The "Export" button will prompt you for a location in which to save the settings file. 
+
+Once saved to the file system, you can use the "Share" button to, for example, copy the settings file
+to Google Drive, or email it as an attachment.
```

Comment:
I'd avoid the term file system (see e.g. https://www.pcgamer.com/students-dont-know-what-files-and-folders-are-professors-say/). I think we can just drop the "to the file system" part of the sentence.

For the example I'd stay away from Google Drive, because Google Drive accounts also show up as an option in the system's "save to" screen. So users can directly save to Google Drive, rather than having to save locally, then share to Google Drive.